packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1971,36 +1971,22 @@ export class LitNodeClientNodeJs
   };
 
   /**
-   * Get session signatures for a set of resources
+   * This is one of the methods that can be used to get session signatures. 
```

Comment:
Maybe change this to:

`Get session signatures for a set of [Lit resources](https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/types_src.ILitResource.html#resource)`